ouroboros-consensus/src/Ouroboros/Consensus/Network/NodeToNode.hs

```diff
@@ -477,6 +501,34 @@ mkApps kernel Tracers {..} Codecs {..} genChainSyncTimeout Handlers {..} =
         channel
         (txSubmissionServerPeerPipelined (hTxSubmissionServer version them))
 
+    aKeepAliveClient
+      :: NodeToNodeVersion
+      -> remotePeer
+      -> Channel m bKA
+      -> m ((), Maybe bKA)
+    aKeepAliveClient _version _them _channel = do
+      labelThisThread "KeepAliveClient"
+      -- TODO implement the client
```

Comment:
What's the plan for this client?

Reply:
The client is being developed, https://github.com/input-output-hk/ouroboros-network/pull/2392 but it is dependent on improvements to the DeltaQ module before it can be enabled.